app/user/cached-user-resolver.js

```diff
@@ -0,0 +1,10 @@
+const userResolver = require('./user-resolver');
+const userInfoCache = require('../cache/cache-config').userInfoCache;
+const jwtUtil = require('../util/jwt');
+
+const getCachedUserDetails = (jwt) => {
+  return userInfoCache.get(jwtUtil.getJwt(jwt), 
```

Comment:
I'd move 
`jwtUtil.getJwt(jwt)`
inside the user info cache get method. 
Imagine we had multiple clients using the user cache all of them would need to remember to call `jwtUtil.getJwt(jwt)` otherwise there could be bugs. Also it would be duplicated code.

Reply:
No need for changes following discussion - both the cached and default resolver both handle with/without the Bearer prefix.